cases/forms/case.py

```diff
@@ -138,10 +150,34 @@ def clean_type(self):
         type = ActionType.objects.get(id=type)
         return type
 
+    def clean(self):
+        if self._is_time_in_future():
+            raise ValidationError("The time of the action can't be in the future.")
```

Comment:
Change this to a curly apostrophe and then you won't need to specially escape the test :)

Reply:
Sorry I didn't manage to follow this one, could you maybe sketch it out in code?

Reply:
Sorry, I meant have `The time of the action can’t be in the future.` with a curly apostrophe here, and then in your test you wouldn't have to use/call `html.escape`.